sympy/series/ap.py

```diff
@@ -0,0 +1,10 @@
+bananas =['1','2','3','4','5']
+
+def is_ap(parame):
+    orange = parame[1] - parame[0]
+    for index in range(len(parame) - 1):
+        if not (parame[index + 1] - parame[index] == orange):
+             return False
+    return True
+
+print(is_ap(bananas))
```

Comment:
I think you should include in a test file. Do not include `print` statement.

Reply:
line 6 can have a one-off error.
change line 5 to for index in range(len(parame) - 2)

This will avoid the one-off error but I don't know enough to guide the PR.T